src/supervisor/watchers/handlers/cron-job.ts

```diff
@@ -137,7 +91,7 @@ export async function cronJobWatchHandler(
 async function isNamespacedCronJobSupportedWithClient(
```

Comment:
I don't think we need this function at all now, CronJobs should always be supported